src/shared/pipes.tsx

```diff
@@ -36,3 +36,16 @@ export const fullNamePipe: Function = (user: User): string => {
 export const booleanPipe: Function = (bool: boolean): string => {
   return bool ? 'YES' : 'NO';
 };
+
+// Formats an array of objects into a string that is a list
+export const listPipe: Function = <T,>(array: T[], transform: (ele: T) => string): string => {
+  var strArray = array.map(transform);
+  return strArray.join(', ');
```

Comment:
```suggestion
  return array.map(transform).join(', ');
```
Pretty sure this can be shortened with chaining since `array.map()` returns a new array.